hedyorg/hedy
# 🪲 Viewing programs shouldn't set programmers mode in editor

Prevents `localStorage` to be modified when enforcing developers mode. This means that when going to a page like view programs, you can return to the code page and not have the programmers mode enforced on you. I also sneaked in a minor fixed regarding not showing the checkmarks on the next and previous adventure button.


Fixes #5771
**How to test**
* Go to the explore page and open a program
* Go back to the code page. Programmers mode should be disabled.
